spec/helpers/sellers_helper_spec.rb

```diff
@@ -75,6 +82,11 @@
         # Create a donation of $10
         item_donation2 = create(:item, seller: seller)
         create(:donation_detail, item: item_donation2, amount: 10_00)
+        expected_seller['donation_amount'] = 21000
```

Comment:
Why are you updating expected_seller here?

Reply:
my bad, moved it to the correct spot in the test, thanks for catching that!